README.md

```diff
@@ -1,7 +1,9 @@
 # Datadog to Terraform Converter
-Converts Datadog monitor JSON into Terraform alarm code. [Install it here.](https://chrome.google.com/webstore/detail/datadog-to-terraform-conv/lafmglpipgongjmbbjngmboifpaodemk)
+Converts Datadog monitor JSON into Terraform alarm code.
 
-### How to Use
+## Installation
+[Chrome Extension](https://chrome.google.com/webstore/detail/datadog-to-terraform-conv/lafmglpipgongjmbbjngmboifpaodemk) | [Firefox Add-on](https://addons.mozilla.org/en-US/firefox/addon/datadog-to-terraform-converter/)
```

Comment:
The firefox link won't work yet as its still uploading to their store. But they've provided this link for when it does go live